test/Sentry.Unity.Tests/IntegrationTests.cs

```diff
@@ -189,27 +189,33 @@ public IEnumerator BugFarmScene_MultipleSentryInit_SendEventForTheLatest()
         {
             yield return SetupSceneCoroutine("1_BugFarm");
 
+            // We should use the sample Dsn for the nextDsn
+            // to avoid static dsn.
+            var options = AssetDatabase.LoadAssetAtPath(ScriptableSentryUnityOptions.GetConfigPath(ScriptableSentryUnityOptions.ConfigName),
+                typeof(ScriptableSentryUnityOptions)) as ScriptableSentryUnityOptions;
+
             var sourceEventCapture = new TestEventCapture();
             var sourceDsn = "https://94677106febe46b88b9b9ae5efd18a00@o447951.ingest.sentry.io/5439417";
             using var firstDisposable = InitSentrySdk(o =>
-            {
-                o.Dsn = sourceDsn;
-                o.AddIntegration(new UnityApplicationLoggingIntegration(eventCapture: sourceEventCapture));
-            });
+             {
+                 o.Dsn = sourceDsn;
+                 o.AddIntegration(new UnityApplicationLoggingIntegration(eventCapture: sourceEventCapture));
+             });
```

Comment:
```suggestion
            });
```
Curious, is the formatter not picking those up?

Reply:
he's the one who messed up :D 
He committed too fast and blocked my previous push. Fixing it now